package.json

```diff
@@ -5,5 +5,43 @@
   "description": "Core HM Platform Enterprise Kit and UI",
   "repository": "https://github.com/humanmade/hm-platform",
   "author": "Human Made Limited",
-  "license": "GPL-3.0"
+  "license": "GPL-3.0",
+  "plugins": {
+    "aws-ses-wp-mail": {
```

Comment:
I can do this in another PR but we'll need the extra info that we'll show in the EK admin screen. Just wanted to check it was intentionally left out at this stage.

Reply:
Yes, it was intentionally left out. I wanted to keep this small on purpose, and there are other tickets for these issues, so new PRs make sense.